packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/android/remotesignout.md

```diff
@@ -2,25 +2,65 @@
 
 Signing out of Okta requires the app to open a browser and navigate to the [end session endpoint](https://developer.okta.com/docs/reference/api/oidc/#logout). Okta destroys the user's session and immediately redirects back to your application.
 
-To do this, you must define a callback route for the sign-out process. 
-
-`TODO: Instructions to add signout redirect URI to gradle file`
+To do this, you must define a callback route for the sign-out process, which you should have done in the [Define a callback route](https://developer.okta.com/docs/guides/sign-into-mobile-app/android/define-callback/) step.
```

Comment:
Somehow I gave the wrong URL, so am fixing:

To do this, you must define a callback route for the sign-out process, which you should have done in the [Define a callback route](https://developer.okta.com/docs/guides/sign-users-out/android/define-callback/) step.

Reply:
Hi @susanharper-okta the url seems correct. defining callback should be done in sign-in section.
The url doesn't exist:
https://developer.okta.com/docs/guides/sign-users-out/android/define-callback/

Reply:
it doesn't exist YET - that's the new section that we added. :)